hydrate.py

```diff
@@ -72,6 +77,40 @@ def get_deployments_for_all_namespaces():
     verbose_print = print if args.verbose else lambda *a, **k: None
     verbose_print("Printing verbosely...")
 
-    config.load_kube_config('kubeconfig')
-    get_pods_for_all_namespaces()
-    get_deployments_for_all_namespaces()
+    # Connect to Cluster

```

Comment:
The main method is a little long, consider extracting some functionality to other methods, for example connect_to_cluster() etc.